[2024-10-04 23:33:16,484] {processor.py:153} INFO - Started process (PID=97037) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:33:16,486] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:33:16,488] {logging_mixin.py:115} INFO - [2024-10-04 23:33:16,487] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:33:16,535] {logging_mixin.py:115} INFO - [2024-10-04 23:33:16,534] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:33:16,536] {logging_mixin.py:115} INFO - [2024-10-04 23:33:16,536] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:33:16,536] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:33:16,563] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2024-10-04 23:34:22,458] {processor.py:153} INFO - Started process (PID=97081) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:34:22,461] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:34:22,462] {logging_mixin.py:115} INFO - [2024-10-04 23:34:22,462] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:34:22,479] {logging_mixin.py:115} INFO - [2024-10-04 23:34:22,479] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:34:22,480] {logging_mixin.py:115} INFO - [2024-10-04 23:34:22,479] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:34:22,480] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:34:22,492] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-10-04 23:35:22,070] {processor.py:153} INFO - Started process (PID=97126) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:35:22,082] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:35:22,083] {logging_mixin.py:115} INFO - [2024-10-04 23:35:22,083] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:35:22,124] {logging_mixin.py:115} INFO - [2024-10-04 23:35:22,123] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:35:22,125] {logging_mixin.py:115} INFO - [2024-10-04 23:35:22,125] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:35:22,125] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:35:22,155] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2024-10-04 23:36:57,250] {processor.py:153} INFO - Started process (PID=98294) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:36:57,295] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:36:57,295] {logging_mixin.py:115} INFO - [2024-10-04 23:36:57,295] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:36:57,316] {logging_mixin.py:115} INFO - [2024-10-04 23:36:57,315] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:36:57,316] {logging_mixin.py:115} INFO - [2024-10-04 23:36:57,316] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:36:57,316] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:36:57,665] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.418 seconds
[2024-10-04 23:38:50,111] {processor.py:153} INFO - Started process (PID=99163) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:38:50,113] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:38:50,114] {logging_mixin.py:115} INFO - [2024-10-04 23:38:50,114] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:38:50,151] {logging_mixin.py:115} INFO - [2024-10-04 23:38:50,150] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:38:50,152] {logging_mixin.py:115} INFO - [2024-10-04 23:38:50,152] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:38:50,152] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:38:50,182] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2024-10-04 23:39:42,044] {processor.py:153} INFO - Started process (PID=99207) to work on /home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:39:42,660] {processor.py:641} INFO - Processing file /home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:39:42,662] {logging_mixin.py:115} INFO - [2024-10-04 23:39:42,662] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:39:42,722] {logging_mixin.py:115} INFO - [2024-10-04 23:39:42,721] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:39:42,723] {logging_mixin.py:115} INFO - [2024-10-04 23:39:42,723] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:39:42,723] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:39:42,749] {processor.py:161} INFO - Processing /home/kennedy/KR/Programação/01.Exercícios/DE/Movies/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.711 seconds
[2024-10-04 23:40:41,309] {processor.py:153} INFO - Started process (PID=99355) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:40:41,330] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:40:41,332] {logging_mixin.py:115} INFO - [2024-10-04 23:40:41,331] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:40:41,371] {logging_mixin.py:115} INFO - [2024-10-04 23:40:41,370] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:40:41,371] {logging_mixin.py:115} INFO - [2024-10-04 23:40:41,371] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:40:41,372] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:40:41,398] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2024-10-04 23:42:02,707] {processor.py:153} INFO - Started process (PID=99626) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:42:03,670] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:42:03,671] {logging_mixin.py:115} INFO - [2024-10-04 23:42:03,671] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:42:03,713] {logging_mixin.py:115} INFO - [2024-10-04 23:42:03,712] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:42:03,714] {logging_mixin.py:115} INFO - [2024-10-04 23:42:03,713] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:42:03,714] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:42:03,752] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.053 seconds
[2024-10-04 23:43:01,946] {processor.py:153} INFO - Started process (PID=99785) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:43:03,119] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:43:03,120] {logging_mixin.py:115} INFO - [2024-10-04 23:43:03,120] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:43:03,156] {logging_mixin.py:115} INFO - [2024-10-04 23:43:03,155] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:43:03,156] {logging_mixin.py:115} INFO - [2024-10-04 23:43:03,156] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:43:03,157] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:43:03,183] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.245 seconds
[2024-10-04 23:43:59,842] {processor.py:153} INFO - Started process (PID=99852) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:00,084] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:44:00,085] {logging_mixin.py:115} INFO - [2024-10-04 23:44:00,085] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:00,127] {logging_mixin.py:115} INFO - [2024-10-04 23:44:00,126] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:44:00,127] {logging_mixin.py:115} INFO - [2024-10-04 23:44:00,127] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:44:00,127] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:00,155] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.319 seconds
[2024-10-04 23:44:15,552] {processor.py:153} INFO - Started process (PID=99872) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:15,693] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:44:15,694] {logging_mixin.py:115} INFO - [2024-10-04 23:44:15,694] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:15,730] {logging_mixin.py:115} INFO - [2024-10-04 23:44:15,729] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:44:15,730] {logging_mixin.py:115} INFO - [2024-10-04 23:44:15,730] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:44:15,730] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:44:15,986] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.441 seconds
[2024-10-04 23:45:48,075] {processor.py:153} INFO - Started process (PID=100889) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:45:48,250] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:45:48,251] {logging_mixin.py:115} INFO - [2024-10-04 23:45:48,251] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:45:48,287] {logging_mixin.py:115} INFO - [2024-10-04 23:45:48,287] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:45:48,288] {logging_mixin.py:115} INFO - [2024-10-04 23:45:48,288] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:45:48,288] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:45:48,314] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.246 seconds
[2024-10-04 23:47:49,608] {processor.py:153} INFO - Started process (PID=101435) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:47:49,656] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:47:49,657] {logging_mixin.py:115} INFO - [2024-10-04 23:47:49,657] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:47:49,695] {logging_mixin.py:115} INFO - [2024-10-04 23:47:49,694] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:47:49,695] {logging_mixin.py:115} INFO - [2024-10-04 23:47:49,695] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:47:49,695] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:47:49,713] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
[2024-10-04 23:50:05,493] {processor.py:153} INFO - Started process (PID=102382) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:50:05,677] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:50:05,678] {logging_mixin.py:115} INFO - [2024-10-04 23:50:05,678] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:50:05,720] {logging_mixin.py:115} INFO - [2024-10-04 23:50:05,719] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:50:05,720] {logging_mixin.py:115} INFO - [2024-10-04 23:50:05,720] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:50:05,721] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:50:05,740] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.253 seconds
[2024-10-04 23:51:56,328] {processor.py:153} INFO - Started process (PID=102765) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:51:56,352] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:51:56,353] {logging_mixin.py:115} INFO - [2024-10-04 23:51:56,353] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:51:56,385] {logging_mixin.py:115} INFO - [2024-10-04 23:51:56,384] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:51:56,385] {logging_mixin.py:115} INFO - [2024-10-04 23:51:56,385] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:51:56,385] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:51:56,399] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2024-10-04 23:53:22,425] {processor.py:153} INFO - Started process (PID=102948) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:53:22,438] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:53:22,438] {logging_mixin.py:115} INFO - [2024-10-04 23:53:22,438] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:53:22,452] {logging_mixin.py:115} INFO - [2024-10-04 23:53:22,452] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:53:22,452] {logging_mixin.py:115} INFO - [2024-10-04 23:53:22,452] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:53:22,453] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:53:22,459] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-10-04 23:55:15,545] {processor.py:153} INFO - Started process (PID=103939) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:55:15,561] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:55:15,562] {logging_mixin.py:115} INFO - [2024-10-04 23:55:15,562] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:55:15,609] {logging_mixin.py:115} INFO - [2024-10-04 23:55:15,608] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:55:15,609] {logging_mixin.py:115} INFO - [2024-10-04 23:55:15,609] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:55:15,610] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:55:15,629] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2024-10-04 23:57:20,302] {processor.py:153} INFO - Started process (PID=105155) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:57:20,310] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:57:20,311] {logging_mixin.py:115} INFO - [2024-10-04 23:57:20,311] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:57:20,355] {logging_mixin.py:115} INFO - [2024-10-04 23:57:20,354] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:57:20,356] {logging_mixin.py:115} INFO - [2024-10-04 23:57:20,355] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:57:20,356] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:57:20,376] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2024-10-04 23:59:18,017] {processor.py:153} INFO - Started process (PID=105365) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:59:19,313] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-04 23:59:19,314] {logging_mixin.py:115} INFO - [2024-10-04 23:59:19,314] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:59:19,342] {logging_mixin.py:115} INFO - [2024-10-04 23:59:19,342] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-04 23:59:19,342] {logging_mixin.py:115} INFO - [2024-10-04 23:59:19,342] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-04 23:59:19,343] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-04 23:59:19,355] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.345 seconds
[2024-10-05 00:01:03,029] {processor.py:153} INFO - Started process (PID=105632) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:01:03,033] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-05 00:01:03,034] {logging_mixin.py:115} INFO - [2024-10-05 00:01:03,034] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:01:03,079] {logging_mixin.py:115} INFO - [2024-10-05 00:01:03,078] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-05 00:01:03,079] {logging_mixin.py:115} INFO - [2024-10-05 00:01:03,079] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-05 00:01:03,079] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:01:03,098] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2024-10-05 00:02:24,142] {processor.py:153} INFO - Started process (PID=105952) to work on /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:02:24,144] {processor.py:641} INFO - Processing file /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-10-05 00:02:24,145] {logging_mixin.py:115} INFO - [2024-10-05 00:02:24,145] {dagbag.py:507} INFO - Filling up the DagBag from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:02:24,189] {logging_mixin.py:115} INFO - [2024-10-05 00:02:24,188] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-05 00:02:24,190] {logging_mixin.py:115} INFO - [2024-10-05 00:02:24,189] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-05 00:02:24,190] {processor.py:653} WARNING - No viable dags retrieved from /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-10-05 00:02:24,209] {processor.py:161} INFO - Processing /home/kennedy/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
